YadavAkhileshh/Alien-Invasion-Defense
# improve preloader

## What does this PR do?

Styling has been applied to the preloader. The preloader is present on all pages of the website with styling.

*Improvements:*

1. *Optimized Preloader:* Enhanced the preloader animation on the index, login/login, and signup/signup pages to improve user experience.

2. *Revamped Preloader Design:* Redesigned the preloader with modern CSS styling to enhance visual appeal.

3. *Dedicated Preloader Script and StyleSheet:* Created a separate JavaScript file named `"preloader.js"` to manage the preloader functionality and CSS file named `" preloaderStyle.css"` to enhance the styling and functionality, ensuring better code organization and maintainability.

Fixes #189 

## Screenshots 📷

**Before changes** 

![Screenshot 2024-10-07 135253](https://github.com/user-attachments/assets/a8a1578b-3f97-4df8-b18d-291886dc3261)


**After adding preloader**


https://github.com/user-attachments/assets/260f08b5-76b1-4834-a47b-b4e368fd5408


## Type of change




- New feature (non-breaking change which adds functionality)

## How should this be tested?

The program was tested locally on a machine using the Chrome browser. To verify the changes, the website was loaded after the modifications were made. This involved manually inspecting the visual elements and functionally testing various user interactions to ensure the changes were implemented correctly.

## Mandatory Tasks
